std/net/curl.d

```diff
@@ -23,7 +23,7 @@ SMTP) )
 )
 
 Note:
-You may need to link to the $(B curl) library, e.g. by adding $(D "libs": ["curl"])
+You may need to link to the $(B curl) library, e.g. by adding `"libs": ["curl"]`
```

Comment:
Now, this is a case where I don't think it should have been `$(D ...)` to begin with - this is json, not D, so it should have been using a generic code syntax. But you can't really automate a decision like that.